dashboard/app/controllers/libraries_controller.rb

```diff
@@ -1,6 +1,10 @@
 class LibrariesController < ApplicationController
-  before_action :require_levelbuilder_mode
-  load_and_authorize_resource find_by: :name
+  before_action :require_levelbuilder_mode, except: :show
+  load_and_authorize_resource find_by: :name, except: :show
```

Comment:
I think everyone is granted permission to read libraries in abilities.rb, so we shouldn't need to skip validation: https://github.com/code-dot-org/code-dot-org/blob/3d41cd121d696d359d79837c7a5550676f9f93fe/dashboard/app/models/ability.rb#L10

Reply:
In this case I think I still need it, to skip the "load" part, so I can manually load from the cache.